ClimsoftVer4/ClimsoftVer4/ucrFormDaily2.vb

```diff
@@ -270,7 +270,8 @@ Public Class ucrFormDaily2
         If bTotalRequired Then
             If ucrInputTotal.IsEmpty AndAlso Not IsValuesEmpty() Then
                 MessageBox.Show("Please enter the Total Value in the [Total] textbox.", "Error in total", MessageBoxButtons.OK, MessageBoxIcon.Error)
-                ucrInputTotal.SetBackColor(Color.Cyan)
+                ucrInputTotal.SetBackColor(Color.Red)
+                ucrInputTotal.GetFocus()
                 bValueCorrect = False
             Else
                 expectedTotal = Val(ucrInputTotal.GetValue)
```

Comment:
At this point, you could have an additional check like 
`expectedTotal = If(ucrInputTotal.IsEmpty, 0, Val(ucrInputTotal.GetValue))`
Though Val() will always return 0 for an empty textbox, I just thought that it will make the code much clearer to show that we are clearly assigning a zero for the case where the textbox is empty

Reply:
We should not assign Zero when the control is empty . we should be be knowing that nothing has been put in. When you you do  `Val(ucrInputTotal.GetValue)` then when the total has nothing it will be zero, which is clearly not what we want

Reply:
I agree. From the code above. When both are empty, we display a message, when any one of them is not empty then we proceed with the calculation. How do we correct this

Reply:
Remember we can have a value empty and have an M, so that should still work out! Or you are not referring to the value entries here.